includes/core-changes/aspnetcore/5.0/blazor-static-web-assets-validation-logic-updated.md

```diff
@@ -0,0 +1,48 @@
+### Blazor: Updated validation logic for static web assets
+
+There was an issue in conflict validation for static web assets in ASP.NET Core 3.1 and Blazor WebAssembly 3.2. The issue:
+
+* Prevented proper conflict detection between the host assets and assets from Razor Class Libraries (RCLs) and Blazor WebAssembly apps.
+* Mostly affects Blazor WebAssembly apps, since by default, static web assets in RCLs are served under the `_content/$(PackageId)` prefix.
+
+#### Version introduced
+
+5.0
+
+#### Old behavior
+
+During development, an RCL's static web assets could be silently overridden with host project assets on the same host path. Consider an RCL that has defined a static web asset to be served at */folder/file.txt*. If the host placed a file at *wwwroot/folder/file.txt*, the file on the server silently overrode the file on the RCL or Blazor WebAssembly app.
+
+#### New behavior
+
+ASP.NET Core correctly detects when this issue happens. It informs of the conflict so that users can take the appropriate action.
```

Comment:
```suggestion
ASP.NET Core correctly detects when this issue happens. It informs you, the user, of the conflict so that you can take the appropriate action.
```